application/routes/audio.py

```
@@ -35,22 +35,27 @@
     if reason != '':
+        logger.warning("Audio: Failed to create audio connection: {}".format(reason))
         abort(403, description=reason)
 
     status, reason = audio.launch_audio()
     if status is False:
+        logger.error("Audio: Audio launch failed: {}".format(reason))
         abort(403, description=reason)
 
     result = {
         'port': AUDIO_PORT,
         'audio_id': audio.id
```

Ruff

Trailing comma missing

Add trailing comma

(COM812)